### DISCUSSION TITLE
Get Select options length after search

### DISCUSSION THREAD

**Discussion by IonelLupu** [ant-design/ant-design#40667] [Q&A]
Is there a way to get the number of displayed options after searching in a Select box? I would like to automatically select the only option available if there is only one visible after a search is made.

For example, I have this select box with phone prefixes:
![image](https://user-images.githubusercontent.com/4083652/217854361-67178be1-c3d4-408e-9ff0-9c6dc9afe1b9.png)

When the user searches for "351" for example, I would like to automatically set the value of the select box to that only option available: 
![image](https://user-images.githubusercontent.com/4083652/217854679-a1a700ce-4582-417e-9606-a1b4ee2a98e3.png)

How can we achieve something like this?

**Comment by Yuiai01** [comment]
@IonelLupu Hello, you can use the select component, see this link https://codesandbox.io/s/tcxdyd I hope I can help you.

**Reply by IonelLupu** [reply]
I am already using the select component. I just need to find a way to get the number of filtered results after searching.

**Reply by Yuiai01** [reply]
> I am already using the select component. I just need to find a way to get the number of filtered results after searching.

It might be possible to store the number of eligible options as a variable when using filterOption, and then render this number with other components. 🤔🤔
